tests/Fixer/Operator/BinaryOperatorSpacesFixerTest.php

```diff
@@ -1643,6 +1643,703 @@ public function testFixAlignDoubleArrow(string $expected, ?string $input = null)
     }
 
     public static function provideAlignDoubleArrowCases(): array
+    {
+        return [
```

Comment:
The git diff is not great but I've duplicated provideAlignDoubleArrowCases into provideAlignScopedDoubleArrowCases with small diff I'll list.